PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -162,64 +163,44 @@ function transformReports(inputReportFiles: string[], index: number)
                 reportType = ReportType.XML_XUNIT;
 
             } else if (node.name == 'CodingStandards') {
-                bStaticAnalysisResult = true;
+                bParsingStaticAnalysisResult = true;
 
             } else if (node.name == 'Rule') {
                 let ruleId = node.attributes.id;
                 let analyzerId = node.attributes.analyzer;
                 if (!bLegacyReport) {
-                    if(!ruleAnalyzerPairs.get(ruleId) && isDtpSettingsValid) {
```

Comment:
It's not necessary to check the existence of ruleId because a Rule in the report won't be duplicated.